src/shims/runtime/logger.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+function getLogger () {
+  return {
+    debug: function (...args) {
+      console.debug(...args)
+    },
+    info: function (...args) {
+      console.info(...args)
+    },
+    warn: function (...args) {
+      console.warn(...args)
+    },
+    error: function (...args) {
+      console.error(...args);
+    }
+  };
+}
```

Comment:
No strong opinion here, but could you return `console` since the API is identical?

Reply:
Sure, was what I did originally, but thought I might more control and perhaps add some formatting and the like, but sure, console might just as well work I suppose.